config.js

```diff
@@ -393,6 +393,12 @@ var config = {
         // The Amplitude APP Key:
         // amplitudeAPPKey: '<APP_KEY>'
 
+        // Configuration for the rtcstats server:
+        // rtcstatsEnabled: true,
+        // rtcstatsEndpoint: wss://rtcstats-server-pilot.jitsi.net/,
+        // By not providing a poll interval rtcstats won't send any getstats data.
```

Comment:
Instead of requiring the interval can we just assume the default of 1000? Also we could simplify the config by treating the presence of `rtcstatsEndpoint` as RTC stats being enabled and the lack of thereof as disabled.

Reply:
Sure, will change.

Reply:
can you update the comment as well, please? it says "By not providing a poll interval rtcstats won't send any getstats data"

Reply:
Aw forgot about the 1000 ms thing, changed and updated the comment.